enterprise/api/utils.py

```diff
@@ -0,0 +1,19 @@
+"""
+Utility functions for the Enterprise API.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django.conf import settings
+
+
+WORKER_USERNAMES = (
```

Comment:
are these `WORKER` usernames, `SERVICE` usernames, or `SERVICE_WORKER` usernames?  I know it's semantics, but might as well be clear on what we're dealing with here.